travelpal/assets/js/components/Main.jsx

```diff
@@ -52,6 +53,10 @@ export default function Main({form, friends, trips}) {
 
 Main.propTypes = {
   form: PropTypes.object.isRequired,
+  bookedForm: PropTypes.object.isRequired,
   friends: PropTypes.array.isRequired,
-  trips: PropTypes.array.isRequired
+  travelDates: PropTypes.array.isRequired,
+  bookedTrips: PropTypes.array.isRequired,
```

Comment:
there is something called shapeOf or arrayOf if you want to use it. I know this isnt like for a real job so idk how detailed we want to be. I don't wanna be the one scrutinizing everyone if we're just okay with gluing things together to get this done with.

Reply:
Yeah, I read about those, but I think it's okay to just leave it like this for the scale of this project. It's already kind of a pain doing the PropTypes as is.

Reply:
No need to the do the props types then. Idk I thought with the PR process we wanted real reviews and stuff

Reply:
No, your reviews are good! I just meant, functionality bugs should be what prevents us from merging to master. But if we manage to have time leftover, we can do one big clean-up PR haha.